app/models/comment.rb

```diff
@@ -1,5 +1,26 @@
 # frozen_string_literal: true
 
 class Comment < ApplicationRecord
+  include GoogleCloudPlatformAccessAuthentication
+
   belongs_to :account
+  before_save :save_sentiment_analysis
+
+  # bodyの内容をもとにGCPで用意されているNLPapiで感情分析を行いその値をDBに格納するメソッド
+  # HACK: fatモデルの原因になりそう
+  def save_sentiment_analysis
+    client = Authorization.init
+
+    body = read_attribute(:body)
+    document = { content: body, type: :PLAIN_TEXT }
+    response = client.analyze_sentiment document: document
+
+    sentiment = response.document_sentiment
+
+    score = sentiment.score.to_f.round(1)
+    magnitude = sentiment.magnitude.to_f.round(1)
+
+    write_attribute(:score, score)
+    write_attribute(:magnitude, magnitude)
```

Comment:
説明変数は全然問題ないのでいいとは思いますが、
fatモデルになるのが心配ならscoreとmagnitudeが保存されるのがキーワード引数(?)でわかるのでこれでもいいのかもしれないです。
```suggestion
    write_attribute(:score, sentiment.score.to_f.round(1))
    write_attribute(:magnitude, sentiment.magnitude.to_f.round(1))
```

Reply:
@sohey-dr 
というのは本来はこういう感情分析の処理は処理でクラス切り分けた方がいいんかな〜と思ったんよ

Reply:
@kishida-bg 
んー、別に他の人からレビューされた訳じゃない自分の意見なんで聞き流してもらっていいんですけど
変更を加えやすくするならドキュメント通りのメソッドの切り分け方でいいと思うんですよね。この処理全てgemに依存している処理なので。

Reply:
@kishida-bg 
微妙ですね、、
Rails way的にはなんとも言えなそう

Reply:
モジュールとしてMix-inできるようになっちゃっている訳ですし

Reply:
今度でいいんですけど、roundの引数の1はコメントで説明するか、変数にしてもいいかも、、